lib/jira/commands/install.rb

```diff
@@ -54,6 +71,15 @@ def token
         io.ask("JIRA token:")
       end
 
+      def cookie
+        response = auth_api.post('session',
```

Comment:
We can just do this
`response = auth_api.post('session', params: session_params)`
Otherwise, we can use:
```
response = auth_api.post(
  'session',
  params: session_params
)
```
or
```
response = auth_api.post(
  'session',
  { params: session_params }
)
```